runtimes/nodejs/actions/initialHtml.js

```diff
@@ -33,12 +62,11 @@ function getHtml(theSignedUrlPut, theSignedUrlGet) {
         <h4> Before uploading a file:</h4>
         <ul>
             <li>Create Cloud Object Storage HMAC Credentials ({hmac:true} on credential creation)</li>
-            <li>Create a bucket on Cloud Object Storage</li>
-            <li>Add the bucket name as a parameter to the get-signed-url action in the cloud-object-storage package</li>
-            <li>Add CORS policy to bucket</li>
+            <li>Create a bucket in Cloud Object Storage Service</li>
+            <li>Add the bucket name as a parameter to this action</li>
         </ul>
         <h4> Current Profile Image:</h4>
-        <img src="http://via.placeholder.com/200x200" class="my-image" style="max-width: 200px; height: auto;"></img>
+        <img src="https://via.placeholder.com/200x200" class="my-image" style="max-width: 200px; height: auto;"></img>
         <h4> Upload a file:</h4>
```

Comment:
Is it ok to be using this link?
What if it changes, etc?

Reply:
I think it should be okay for now.  I'm not sure of another good/simple way to get an image to hold the place.  If the link changes, it's low risk -- will just show up with a broken image instead of a placeholder.